src/qml/pages/node/PeerDetails.qml

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2024 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import org.bitcoincore.qt 1.0
+import "../../controls"
+import "../../components"
+
+Page {
+    signal backClicked()
+
+    property PeerDetailsModel details
+
+    Connections {
+        target: details
+        function onDisconnected() {
+            root.backClicked()
+        }
+    }
+
+    id: root
+    background: null
+
+    header: NavigationBar2 {
+        leftItem: NavButton {
+            iconSource: "image://images/caret-left"
+            text: qsTr("Back")
+            onClicked: root.backClicked()
+        }
+        centerItem: Header {
+            headerBold: true
+            headerSize: 18
+            header: qsTr("Peer " + details.nodeId)
+        }
+    }
+
+    component PeerKeyValueRow: Row {
+        width: parent.width
+        property string key: ""
+        property string value: ""
+        CoreText {
+            color: Theme.color.neutral9;
+            text: key;
+            width: 125;
+            horizontalAlignment: Qt.AlignLeft;
+        }
+        CoreText {
+            color: Theme.color.neutral9;
+            elide: Text.ElideRight;
+            wrapMode: Text.WordWrap;
+            text: value
+            width: parent.width - 125;
+            horizontalAlignment: Qt.AlignLeft;
+        }
+    }
+
+    ScrollView {
+        id: scrollView
+        width: parent.width
+        height: parent.height
+        clip: true
+        contentWidth: width
+
+        Column {
+            width: Math.min(parent.width - 40, 450)
+            anchors.horizontalCenter: parent.horizontalCenter
+            spacing: 10
```

Comment:
we want to add top and bottom padding of 30 to adhere to the design file

```suggestion
            spacing: 10
            topPadding: 30
            bottomPadding: 30
```

Reply:
See here, this starts to look like the design file
| master | patch |
| ------ | ----- |
| <img width="708" alt="top-bottom-padding-master" src="https://github.com/bitcoin-core/gui-qml/assets/23396902/d4705cd2-14b5-4ca0-a5fd-5847c7b351fd"> | <img width="752" alt="top-bottom-padding-patch" src="https://github.com/bitcoin-core/gui-qml/assets/23396902/0eb9f79b-7084-4b5e-a942-ee907a1beee3"> |



Design file screenshot of top padding:

<img width="778" alt="Screenshot 2024-02-13 at 10 30 16 PM" src="https://github.com/bitcoin-core/gui-qml/assets/23396902/927f28a0-c18b-4b7c-b91c-6d7ea5e69fa6">